### ISSUE TITLE
[Fury Warrior] Whirlwind keeps getting suggested

### ISSUE THREAD

**Issue by RussellAnas** [herotc/hero-rotation#754]
Sometimes during rotation whirlwind keeps getting suggested even when only 1 melee spell has been cast using the cleave buff. Ideally it should be suggested only once both instances of the buff have been used.

**Comment by RussellAnas** [comment]
Any update on this? It's still happening. Whirlwind gets suggested even when stacks of whirlwind (cleave) buff are remaining. This happens even on 2 enemies so doesn't seem like it does that because of possible higher dps.

**Comment by Cilraaz** [comment]
Whirlwind is a fallback when there's nothing else to cast. Please enable Debug Output and let me know what is text is displayed when you believe Whirlwind should not be suggested.

**Comment by RussellAnas** [comment]
In this image, if you see that after every whirlwind, there were 2 single target spells suggested. At the last one, I did whirlwind, and then it suggested one ST spell, then back to whirlwind. Even though I had full rage for Rampage. And it's impossible to think of a scenario where whirldwind will actually take more important when Rampage is up on a cleave buff
![image](https://user-images.githubusercontent.com/77004545/116815213-f54e3a00-ab75-11eb-9b14-17ffab8c2eed.png)

**Comment by RussellAnas** [comment]
I also tested one more thing. This seems to occur a lot more when I've changed specs from Arms to Fury?

**Comment by Cilraaz** [comment]
This should have been fixed in the commit on June 7th (https://github.com/herotc/hero-rotation/commit/796c62ff9c72bc6b17e88c908acadd57c7e0315e#diff-c8dc33585a3f0b842f25329738b635b6d7946931ce1ba90923f23dec2d9e4ec2). No reports of issues since. Closing this issue. If there is still a problem, please feel free to open a new issue.
